src/en/authors-charm-metadata.md

```diff
@@ -104,5 +104,45 @@ and whether it's designed for deployment as a
   - if the charm is subordinate, it must contain at least one `requires`
     relation with container scope.
 
+`resources` allows you to add a blob that your charm can utilize.
```

Comment:
Taken from http://www.jorgecastro.org/2016/04/19/a-quick-intro-to-juju-resources/